internal/service/operation/message_test.go

```
@@ -116,9 +120,10 @@
 						},
 					},
 				},
-				uow: &mockUnitOfWork{
-					execChan: make(chan struct{}),
-				},
+			},
+			setupMocks: func(t *testing.T, mockUow *mocks.MockunitOfWork, mockDriver *storagemocks.MockDriver) {
+				t.Helper()
+
```

golangci-lint (2.5.0)

[error] unnecessary whitespace (trailing-whitespace)

(wsl_v5)